src/config/navigation/defaultStackConfig.tsx

```diff
@@ -343,6 +343,26 @@ export const defaultStackConfig = ({
       screenComponent: SettingsScreen,
       screenOptions: { title: texts.screenTitles.settings }
     },
+    {
+      routeName: ScreenName.SUEList,
+      screenComponent: IndexScreen,
+      initialParams: {
+        title: texts.screenTitles.sue.listView,
+        query: QUERY_TYPES.SUE.REQUESTS,
+        screenType: 'list',
+        usedAsInitialScreen: true
+      }
+    },
+    {
+      routeName: ScreenName.SUEMap,
+      screenComponent: IndexScreen,
+      initialParams: {
+        title: texts.screenTitles.sue.mapView,
+        query: QUERY_TYPES.SUE.REQUESTS,
+        screenType: 'map',
```

Comment:
what about passing the component directly here, `component: SueMapScreen` or `component: SueListScreen`, instead of having a complex logic with conditions later in `SueIndexScreen`?

Reply:
Absolutely right! That way we can avoid more complexity.
https://github.com/smart-village-solutions/smart-village-app-app/pull/960/commits/754a44edc67676d05ca82e810e413c9d9a8c8e77